# Lab 2 - Работа с данными запроса
## Custom
```python
# ./app/app.py

"""
    Данная лабораторная работа запущена на: 
    - внутреннем порту [5002] 
    - внешнем порту [5002]
"""
# ...
if __name__ == '__main__':
    app.run(debug=True, port=5002)  # Customize app port 
```

## Run
```bash
python3 run ./app/app.py
```

## Notes
- [X] Отображение данных запроса
- [X] Форма с обработкой ошибок

> Из-за проблем с OpenVpn я использовала в качестве хостинга личный сервер
